cloudant-sync-datastore-core/src/test/java/com/cloudant/sync/internal/replication/AttachmentsPullTest.java

```diff
@@ -274,8 +267,24 @@ private void updateRevisionAndAttachment() {
                 attachmentData2.getBytes());
 
         // putting attachment will have updated the rev
-        bar = remoteDb.get(Bar.class, res.getId());
-        rev = bar.getRevision();
+        rev = getLatestRevision(id, rev);
+    }
+
+    /**
+     * Utility for reading writes. When we are writing attachments it might take a little time for
+     * new revision to appear so we validate that the rev has incremented from what we currently
+     * know about before deciding that we have the latest rev.
+     * @param docId
```

Comment:
Should probably document the params here, or remove the `@param` and `@return` annotations.